product/views/MiddlewareDeployment.yaml

```diff
@@ -0,0 +1,74 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: Middleware Deployments
+
+# Menu name
+name: Middleware Deployments
+
+# Main DB table report is based on
+db: MiddlewareDeployment
+
+# Columns to fetch from the main table
+cols:
+- name
+
+
+# Included tables (joined, has_one, has_many) and columns
+include:
+  middleware_server:
+    columns:
+    - name
+
+# Included tables and columns for query performance
+
+
+# Order of columns (from all tables)
+col_order:
+- name
+- middleware_server.name
+
+# Column titles, in order
+headers:
+- Deployment Name
+- Server
+
+# Condition(s) string for the SQL query
+conditions:
+
+# Order string for the SQL query
+order: Ascending
+
+# Columns to sort the report on, in order
+sortby:
+- name
+- middleware_server.name
```

Comment:
I think this needs an entry in includes_for_find...but they section might be dynamic... @martinpovolny ?  I can't remember anymore :sweat_smile: 

In the next report, it's specified, so either way, we should try to be consistent.